builtin/logical/pki/path_tidy.go

```diff
@@ -113,6 +119,13 @@ func pathTidy(b *backend) *framework.Path {
 func pathTidyCancel(b *backend) *framework.Path {
 	return &framework.Path{
 		Pattern: "tidy-cancel$",
+
+		DisplayAttrs: &framework.DisplayAttributes{
+			OperationPrefix: operationPrefixPKI,
+			OperationVerb:   "tidy",
```

Comment:
Maybe cancel should be the verb?

Reply:
I was trying to keep the 3 somewhat consistent for easier search/auto-completion in IDEs. But I wouldn't mind `pki-cancel-tidy` as well.

| Path | OperationIDs |
|------|--------------|
| "/{pki_mount_path}/tidy" |        pki-tidy |
| "/{pki_mount_path}/tidy-cancel" |        pki-tidy-cancel |
| "/{pki_mount_path}/tidy-status" |        pki-tidy-status |